src/main/resources/session-template-data/session-data.csv

```diff
@@ -1,165 +1,165 @@
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,15:30:00,16:30:00,2023-03-10,,TUESDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,14:00:00,15:00:00,2023-03-10,,WEDNESDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,15:30:00,16:30:00,2023-03-10,,WEDNESDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,14:00:00,15:00:00,2023-03-10,,FRIDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,15:30:00,16:30:00,2023-03-10,,FRIDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,14:00:00,15:00:00,2023-03-10,,SATURDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,15:30:00,16:30:00,2023-03-10,,SATURDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,14:00:00,15:00:00,2023-03-10,,SUNDAY,FALSE,,
+EWI,Visits Main Hall,SOCIAL,10,2,FALSE,15:30:00,16:30:00,2023-03-10,,SUNDAY,FALSE,,
 % DHI Prison
-DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,TUESDAY,FALSE,,,
-DHI,Visits Main Hall,SOCIAL,24,3,FALSE,09:30:00,11:30:00,2023-03-10,,SATURDAY,FALSE,,,
-DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,SATURDAY,FALSE,,,
-DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,SUNDAY,FALSE,,,
+DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,TUESDAY,FALSE,,
+DHI,Visits Main Hall,SOCIAL,24,3,FALSE,09:30:00,11:30:00,2023-03-10,,SATURDAY,FALSE,,
+DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,SATURDAY,FALSE,,
+DHI,Visits Main Hall,SOCIAL,24,3,FALSE,13:30:00,15:45:00,2023-03-10,,SUNDAY,FALSE,,
 % MHI Prison
-MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-03-10,,TUESDAY,FALSE,,,
-MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-03-10,2023-04-05,THURSDAY,FALSE,,,
-MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:30:00,16:15:00,2023-03-10,,SATURDAY,FALSE,,,
-MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:30:00,16:15:00,2023-03-10,,SUNDAY,FALSE,,,
+MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-03-10,,TUESDAY,FALSE,,
+MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-03-10,2023-04-05,THURSDAY,FALSE,,
+MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:30:00,16:15:00,2023-03-10,,SATURDAY,FALSE,,
+MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:30:00,16:15:00,2023-03-10,,SUNDAY,FALSE,,
 % MHI Prison skip april the 6th
-MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-04-07,,THURSDAY,FALSE,,,
+MHI,Visits Main Hall,SOCIAL,14,2,FALSE,13:15:00,16:00:00,2023-04-07,,THURSDAY,FALSE,,
 % BNI Prison
-BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,TUESDAY,FALSE,,,
-BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,WEDNESDAY,FALSE,,,
-BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,THURSDAY,FALSE,,,
-BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,SATURDAY,FALSE,,,
-BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,SUNDAY,FALSE,,,
\ No newline at end of file
+BNI,Visits Main Hall,SOCIAL,60,5,FALSE,14:15:00,15:45:00,2023-03-10,,TUESDAY,FALSE,,
```

Comment:
not part of this review but please update time to new BNI values..

Reply:
yes, need to do that,  will do that as part of a separate PR.

Reply:
okay will approve with that in mind